src/test/kotlin/TestProject/TestHelloWorld.kt

```diff
@@ -0,0 +1,12 @@
+package TestProject
```

Comment:
This file still needs to be deleted. Maybe you've renamed it locally and only commited the addition, not the deletion?